### PR TITLE
ContextCall

### PR DESCRIPTION
**Pull request by ibeckermayer**

### CODE REVIEW

File: crates/ironrdp-rdpdr/src/lib.rs [hunk 1]
```diff
@@ -109,6 +109,7 @@ impl Rdpdr {
             let call = ScardCall::decode(req.io_control_code, payload)?;
 
             debug!(?req);
+            debug!(?req.io_control_code);
             debug!(?call);
```
Comment (CBenoit) [inline crates/ironrdp-rdpdr/src/lib.rs:113]: Nit: to limit pressure on the logger and reduce output size, I think this should be in a single log record
```suggestion
            debug!(?req, ?req.io_control_code, ?call);
```
And I would expect `req.io_control_code` to be included into the Debug representation of `req`. 

That being said, maybe this is generating an output that is easier to exploit for you so I don’t insist on this

**Review by CBenoit — APPROVED** [review]
Looks good to me. I left a nit comment that you may address or not. Tell me if I should merge the PR as-is or wait for you to change something.

File: crates/ironrdp-rdpdr/src/lib.rs (hunk 1 above)
Comment (ibeckermayer) [reply crates/ironrdp-rdpdr/src/lib.rs]: Just marginally easier for me to read this way: 93b8c24f4e74d590505e600c87925fb2199c64bd

**Comment by ibeckermayer** [comment]
Ready to merge!

**Pull request merged**